frontend/public/locales/en/public.json

```diff
@@ -1,191 +1,69 @@
 {
-  "API resources": "API resources",
-  "true": "true",
-  "false": "false",
-  "All groups": "All groups",
-  "No group": "No group",
-  "All versions": "All versions",
-  "All scopes": "All scopes",
-  "Cluster": "Cluster",
-  "Namespace": "Namespace",
-  "Kind": "Kind",
-  "Group": "Group",
-  "Version": "Version",
-  "Namespaced": "Namespaced",
-  "Description": "Description",
-  "by kind": "by kind",
-  "API Explorer": "API Explorer",
-  "API group": "API group",
-  "API version": "API version",
-  "Verbs": "Verbs",
-  "Short names can be used to match this resource on the CLI.": "Short names can be used to match this resource on the CLI.",
-  "Short names": "Short names",
-  "Subjects": "Subjects",
-  "Access review": "Access review",
-  "Subject": "Subject",
-  "Type": "Type",
-  "User": "User",
-  "ServiceAccount": "ServiceAccount",
-  "Verb": "Verb",
-  "by subject": "by subject",
-  "The following subjects can {{verb}} {{plural}} in namespace {{ namespace }}": "The following subjects can {{verb}} {{plural}} in namespace {{ namespace }}",
```

Comment:
@rebeccaalpert I'm a bit confused why these strings were removed? It doesn't look like you removed them from source? Were our translation files just out of date?

Reply:
@rebeccaalpert may be contributing to frontend test failures:

```bash
[36mINFO[0m[2021-04-29T19:07:11Z] $ NODE_OPTIONS=--max-old-space-size=4096 yarn eslint . 
[36mINFO[0m[2021-04-29T19:07:11Z] $ eslint --ext .js,.jsx,.ts,.tsx,.json,.gql,.graphql --color . 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m[0m                                     
[36mINFO[0m[2021-04-29T19:07:11Z] [0m[4m/go/src/github.com/openshift/console/frontend/public/locales/en/public.json[24m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m10:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m11:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m12:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m15:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m26:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m27:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m28:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m35:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m37:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m38:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m43:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m83:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m95:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m   [2m99:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m  [2m150:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m  [2m293:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m  [2m416:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m  [2m581:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m  [2m687:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m  [2m744:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m  [2m745:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m  [2m791:4[22m  [31merror[39m  Duplicate object key  [2mjson/*[22m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m[0m                                     
[36mINFO[0m[2021-04-29T19:07:11Z] [0m[31m[1m✖ 22 problems (22 errors, 0 warnings)[22m[39m[0m 
[36mINFO[0m[2021-04-29T19:07:11Z] [0m[31m[1m[22m[39m[0m                  
[36mINFO[0m[2021-04-29T19:07:11Z] error Command failed with exit code 1.       
```

Reply:
I reset the file to the way it was before I made any changes, ran `yarn i18n` again, and got this same output. Not sure what's up.